workbench-core/environments/src/postDeployment/cognitoSetup.ts

```diff
@@ -33,18 +33,7 @@ export default class CognitoSetup {
     const userPoolId = await this.getUserPoolId();
     console.log(`User pool id: ${userPoolId}`);
 
-    // Create IT Admin and Researcher groups in user pool if they do not exist
-    try {
-      await this.createGroup('ITAdmin', userPoolId);
-      console.log('Creating IT Admin group because group does not exist');
-    } catch (e) {
-      if (e instanceof GroupExistsException) {
-        console.log(`IT Admin group already exists in user pool ${USER_POOL_NAME}`);
-      } else {
-        throw e;
-      }
-    }
-
+    // Create Researcher groups in user pool if they do not exist
     try {
       await this.createGroup('Researcher', userPoolId);
```

Comment:
Do we still need to create the researcher role?

Reply:
I opted to keep changes minimal to avoid any potential issues downstream  since I don't have all the context however if removal of Researcher role is safe operation I can delete it.

Reply:
This group can be removed.